SocialCareCaseViewerApi.Tests/V1/Controllers/ResidentControllerTests/UpdateResidentTests.cs

```diff
@@ -35,6 +35,16 @@ public void UpdatePersonReturns200WhenSuccessful()
             result?.StatusCode.Should().Be(204);
         }
 
+        [Test]
+        public void PatchPersonReturns200WhenSuccessful()
```

Comment:
extremely small detail but why is statuscode 200 mentioned in the test name and the status code is actually 204? 🤔

Reply:
Good spot, fixed this and previous test name